test/format-test.cc

```diff
@@ -1577,7 +1577,7 @@ struct explicitly_convertible_to_string_like {
       typename String,
       typename = typename std::enable_if<
         std::is_constructible<String, const char*, std::size_t>::value>::type>
-  explicit operator String() const { return String("foo", 3u); }
+  operator String() const { return String("foo", 3u); }
```

Comment:
`explicit` is required here. Please use `FMT_EXPLICIT` instead.

Reply:
Oh, yeah, this makes sense. Corrected!